spec/models/email_spec.rb

```diff
@@ -72,5 +72,17 @@
         expect(document).to match('Format: Senior Thesis')
       end
     end
+    describe 'bound-with record' do
+      let(:properties) do
+        {
```

Comment:
@sandbergja can you add an id for the solr document? 
`id: 9947055653506421` any id. 

Beyond that If you'd like a boundwith example `id: '9947055653506421', contained_in_s: ['99124977073506421']`
https://catalog.princeton.edu/catalog/99124977073506421/raw, https://catalog.princeton.edu/catalog/9947055653506421/

Reply:
Added an id.  Thanks!